### ISSUE TITLE
how to change canvas width and height full window ?

### ISSUE THREAD

**Issue by asepwaee** [luileito/jsketch#4]
when i set width and height 100% the point of stroke not center with mouse 
![mouse](https://github.com/luileito/jsketch/assets/26277417/d2f1880b-f411-4706-8762-048a7961cd18)

**Comment by luileito** [comment]
Have you tried to set the size (width and height canvas attributes) in absolute pixel values?

**Comment by asepwaee** [comment]
@luileito yes i did it and working fine, but when i change to percent not working. 
i want to make the canvas responsive for every device width. is this packages support for doing that ?

**Comment by luileito** [comment]
This isn't currently supported, but you can do it with JS: https://stackoverflow.com/a/39774800
